integrations/amazon_bedrock/tests/test_chat_generator.py

```diff
@@ -22,12 +22,21 @@
 ]
 
 # so far we've discovered these models support streaming and tool use
-STREAMING_TOOL_MODELS = ["anthropic.claude-3-5-sonnet-20240620-v1:0", "cohere.command-r-plus-v1:0"]
+STREAMING_TOOL_MODELS = [
+    "anthropic.claude-3-5-sonnet-20240620-v1:0",
+    "us.anthropic.claude-sonnet-4-20250514-v1:0",
+    "cohere.command-r-plus-v1:0",
+]
 
 MODELS_TO_TEST_WITH_IMAGE_INPUT = [
     "us.anthropic.claude-sonnet-4-20250514-v1:0",
 ]
 
+MODELS_TO_TEST_WITH_THINKING = [
+    "arn:aws:bedrock:us-east-1::inference-profile/us.anthropic.claude-3-7-sonnet-20250219-v1:0",
```

Comment:
nit: not an AWS expert here, but can the `arn:aws:bedrock:us-east-1::inference-profile` part be avoided?

Reply:
OK to leave it as is. Better to have something that works both locally and in the CI.
Thanks for the explanation.